src/frontend/src/handler/create_connection.rs

```diff
@@ -51,19 +49,10 @@ fn resolve_private_link_properties(
     let provider = get_connection_property_required(with_properties, CONNECTION_PROVIDER_PROP)?;
     let service_name =
         get_connection_property_required(with_properties, CONNECTION_SERVICE_NAME_PROP)?;
-    let availability_zones_str =
-        get_connection_property_required(with_properties, CONNECTION_AVAIL_ZONE_PROP)?;
-    let availability_zones: Vec<String> =
-        serde_json::from_str(&availability_zones_str).map_err(|e| {
-            RwError::from(ProtocolError(format!(
-                "Can not parse {}: {}",
-                CONNECTION_AVAIL_ZONE_PROP, e
-            )))
-        })?;
     Ok(create_connection_request::PrivateLink {
         provider,
         service_name,
-        availability_zones,
+        availability_zones: vec![],
```

Comment:
can I ask why don't we delete this field in the rpc proto since it is deprecated (and nobody is currently using `create connection`..